src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -149,12 +152,6 @@ public void Startup(StartupParams startupParams)
             PackageLoader.RequestLoadNodeLibrary += RequestLoadNodeLibraryHandler;
             PackageLoader.RequestLoadCustomNodeDirectory += RequestLoadCustomNodeDirectoryHandler;
 
-            if (PythonServices.PythonEngineManager.Instance.AvailableEngines.Count == 0)
-            {
-                PythonServices.PythonEngineManager.Instance.LoadDefaultPythonEngine(AppDomain.CurrentDomain.GetAssemblies().
-                                                                                    FirstOrDefault(a => a != null && a.GetName().Name == PythonServices.PythonEngineManager.CPythonAssemblyName));
-            }
-
```

Comment:
For cleaner code, if we are not going to be supporting Python Engine other than `PythonNet3`, then we should remove `PythonEngineManager` and simplify the code surrounding its use.